types/react/experimental.d.ts

```diff
@@ -122,6 +122,6 @@ declare module "." {
         /**
          * @see https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert
          */
-        inert?: boolean | undefined;
+        inert?: '' | boolean | undefined;
```

Comment:
This won't be supported in any React version. It's either `'' | undefined` or `boolean | undefined` but never both.

Reply:
@eps1lon Thanks for the correction. I've added a note in my previous comment.

However, this suggests to me that setting the inert type to `'' | boolean | undefined` is the better choice. My thinking is that `boolean` is a new feature while historically `'' | undefined` worked. So any older code that uses `inert` will use `inert=""` - thus the type for those usages won't need to be updated, which is the purpose of this PR (to allow older code to pass the type checks without being updated). 

I would say this should be set to `undefined | boolean` if this is intentionally introduced as a breaking change by react.

Reply:
From the type checks in CI, it seems that this is possibly an intentional breaking change (assuming from the comment in this ts-expect-error)
```js
<div // @ts-expect-error Old workaround that used to result in `element.inert = true` but would now result in `element.inert = false`
 inert="" />;
```

But this seems rather odd to me to make such an aggressive breaking change - I suppose it is reasonable to have breaking changes between major releases, but it suggests that library authors that use the `inert` property will not be able to support react 19 as well as older react versions (without writing version checks in their code) 🤔